embulk-cli/src/main/java/org/embulk/cli/EmbulkMigrate.java

```diff
@@ -236,7 +237,9 @@ public void copy(String sourceResourcePath, String destinationFileName)
         {
             Path destinationPath = this.basePath.resolve(destinationFileName);
             Files.createDirectories(destinationPath.getParent());
-            Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath);
+            Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath,
+                    StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Yes, but it's okay. I've merged it for the time being.

I was not sure it was the Airlift format...